README.md

```diff
@@ -2,4 +2,12 @@
 
 [![Build Status](https://travis-ci.org/IBM/kube101.svg?branch=master)](https://travis-ci.org/IBM/kube101)
 
-This repository contains stuff. 
+This repository contains introductory material for Kubernetes.
+
+There is a presenter-run [meetup], including [automated scripts] for running a demonstration of Kubernetes as provided by IBM Cloud Kubernetes Service. This should take from 45 minutes to an hour and a half based on the style of the presenter as well as audience participation.
+
+There is a self-guided [workshop], with detailed explanations about the principles of operating an application in a Kubernetes environment. Including some time for setup, this should take between 2-4 hours.
```

Comment:
Minor thing, but this is meant to be both self-guided as well as used in host-run workshops.

Reply:
you can edit it, you know?